### PR TITLE
[IMP] add job helper for simplifying testing

### PR DESCRIPTION
**Pull request by sebastienbeau**
Add a class and a mixing to simplify the test with job
Here is an case of use : https://github.com/akretion/connector-search-engine/blob/36c59f1f773f8ad4578af940c4a3ece78383c4ff/connector_algolia/tests/test_connector_algolia.py#L14

### CODE REVIEW

File: queue_job/tests/common.py [hunk 1]
```diff
@@ -1 +1,42 @@
...
+    @property
+    def count_existing(self):
+        return len(self.existing)
+
+    @property
+    def created(self):
+        return self.all - self.existing
+
+    @property
+    def all(self):
```
Comment (guewen) [inline queue_job/tests/common.py]: `all` is a  builtin ;)

File: queue_job/tests/common.py [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+class JobCounter(object):
+
+    def __init__(self, env):
+        super(JobCounter, self).__init__()
+        self.env = env
+        self.existing = self.search_all()
+
+    def count_all(self):
+        return len(self.all())
```
Comment (guewen) [inline queue_job/tests/common.py]: s/self.all()/self.search_all()/

**Review by guewen — APPROVED** [review]
Looks good!

**Review by simahawk — APPROVED** [review]

**Comment by guewen** [comment]
I'm going to stay offline for some time, if we have enough approvals could you take care of merging @lmignon? Thanks :)

**Review by lmignon — APPROVED** [review]

**Comment by sebastienbeau** [comment]
Merci !

**Pull request merged**